src/bandersnatch/main.py

```diff
@@ -96,6 +96,12 @@ def _verify_parser(subparsers: argparse._SubParsersAction) -> None:
         default=0,
         help="# of parallel iops [Defaults to bandersnatch.conf]",
     )
+    v.add_argument(
+        "--force-sha256-check",
+        action="store_true",
+        default=False,
+        help="force sha256 check on all storage backend",
```

Comment:
Can we say it defaults to on for filesystem backend
```suggestion
        help="Force sha256 check on all storage backends. POSIX filesystems default to checking.",
```